docker-compose.yml

```diff
@@ -110,7 +111,7 @@ services:
       - _APP_OPTIONS_ABUSE
       - _APP_OPTIONS_ROUTER_PROTECTION
       - _APP_OPTIONS_FORCE_HTTPS
-      - _APP_OPTIONS_FUNCTIONS_FORCE_HTTPS
+      - _APP_OPTIONS_COMPUTE_FORCE_HTTPS
```

Comment:
Same comment about the namespace here.

Reply:
Fixed too, `_APP_OPTIONS_ROUTER_FORCE_HTTPS`